src/protocols/secure_channel/CASESession.h

```diff
@@ -157,13 +157,16 @@ class DLL_EXPORT CASESession : public Messaging::UnsolicitedMessageHandler,
 
     FabricIndex GetFabricIndex() const { return mFabricIndex; }
 
+    bool IsPeerActive() { return mExchangeCtxt.HasValue() && mExchangeCtxt.Value()->HasReceivedAtLeastOneMessage(); };
     // Compute our Sigma1 response timeout.  This can give consumers an idea of
     // how long it will take to detect that our Sigma1 did not get through.
-    static System::Clock::Timeout ComputeSigma1ResponseTimeout(const ReliableMessageProtocolConfig & remoteMrpConfig);
+    static System::Clock::Timeout ComputeSigma1ResponseTimeout(const ReliableMessageProtocolConfig & remoteMrpConfig,
+                                                               bool isPeerActive);
```

Comment:
For a _Sigma1_, again if we are trying to match CalculateNextRetransTime(), the peer is never "known active", right?  Because it's the first message on the exchange.

Reply:
i agree, updated, thanks